### DISCUSSION TITLE
Feyncalc not evaluating some functions

### DISCUSSION THREAD

**Discussion by Arain001** [FeynCalc/feyncalc#180] [Q&A]
I installed Feyncalc recently and tried using it for some calculations. For some specific functions, Mathematica is not expanding the function for inputs. I do not know why. Feyncalc is the only package I use on Mathematica.

The FAD function should give the O/P of - i/(pi^2 (q^2-m^2)). Is there any way to resolve this? 

![nex](https://user-images.githubusercontent.com/108520802/176867460-0879f535-77e7-4b5b-a1b5-37d468aa9835.jpeg)

**Comment by vsht** [comment]
I'm not sure that I understand what exactly the problem is. Could you explain a bit what kind of output do you expect?

**Reply by anaik49** [reply]
I am having the same issue. I think it has to do with how or where FeynCalc is installed, but Mathematica doesnt recognize the function names and doesnt evaluate the functions. Am I doing something stupid?
<img width="1132" alt="Screen Shot 2022-11-24 at 3 25 38 PM" src="https://user-images.githubusercontent.com/58065363/203859233-fb93159b-75b6-4acd-b137-fae2acdfe51b.png">

**Reply by vsht** [reply]
Since all FeynCalc symbols are marked blue, FeynCalc is obviously not loaded on your kernel.
